ofrak_core/test_ofrak/components/assets/string_test.c

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Can you add a README or a breadcrumb describing how you built string_test.out? Might be trivial, but it will be helpful down the line for reproducibility.